docs/basics.rst

```diff
@@ -125,7 +125,7 @@ Example Pipfile.lock
 
 - Generally, keep both ``Pipfile`` and ``Pipfile.lock`` in version control.
 - Do not keep ``Pipfile.lock`` in version control if multiple versions of Python are being targeted.
-- Specify your target Python version in your `Pipfile`'s ``[requires]`` section. Ideally, you should only have one target Python version, as this is a deployment tool.
+- Specify your target Python version in your `Pipfile`'s ``[requires]`` section. Ideally, you should only have one target Python version, as this is a deployment tool. ``python_version`` can be ``>`` Python 2.7 and ``<`` Python 3.7.
```

Comment:
This line is confusing, people may think he should define `python_version = ">2.7"` but we do not support this.

We shouldn't constrain the value of `python_version`, it should be working for any Python version that virtualenv supports, and no upper bounds for future python versions.

Reply:
Can you please write what you think you know as proper statement? Did you read the ticket, ticket clearly says the user didn't know the supported Python version numbers, right?

Reply:
I think the point is user doesn't know the valid value format of that field.
For `python_version`, `X.Y` is the proper value, without other binary operators, for `python_full_version`, it is `X.Y.Z`

Reply:
Perfect.